src/Wallabag/CoreBundle/Helper/ContentProxy.php

```diff
@@ -68,6 +69,28 @@ public function updateEntry(Entry $entry, $url, array $content = [], $disableCon
         $this->stockEntry($entry, $content);
     }
 
+    /**
+     * Remove invalid UTF-8 characters from the given string in following steps:
+     * - try to interpret the given string as ISO-8859-1, convert it to UTF-8 and return it (if its valid)
+     * - simply remove every invalid UTF-8 character and return the result (https://stackoverflow.com/a/1433665)
+     * @param String $rawText
+     * @return string
+     */
+    private function sanitizeUTF8Text(String $rawText) {
```

Comment:
Don't type parameter, it comes with PHP 7, wallabag should still be compatible with PHP 5